core/vtk/ttkAlgorithm/ttkAlgorithm.cpp

```diff
@@ -9,6 +9,10 @@
 #include <vtkCellTypes.h>
 #include <vtkCommand.h>
 #include <vtkDataSet.h>
+#include <vtkGenerateGlobalIds.h>
```

Comment:
I would put this include within the `#if TTK_ENABLE_MPI` (it's only used when MPI is enabled)